packages/components/src/z-stack/stories/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+/**
+ * External dependencies
+ */
+import { boolean, number } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import { Elevation } from '../../ui/elevation';
+import { HStack } from '../../h-stack';
+import { View } from '../../view';
+import { ZStack } from '..';
+
+export default {
+	component: ZStack,
+	title: 'Components/ZStack',
+};
+
+const Avatar = () => {
+	return (
+		<View>
+			<View
+				style={ {
+					border: '3px solid black',
+					borderRadius: '9999px',
+					height: '48px',
+					width: '48px',
+					backgroundColor: 'lightgray',
```

Comment:
Related to https://github.com/WordPress/gutenberg/pull/31613/files#r635830223

```suggestion
const Avatar = ( { backgroundColor } ) => {
	return (
		<View>
			<View
				style={ {
					border: '3px solid black',
					borderRadius: '9999px',
					height: '48px',
					width: '48px',
					backgroundColor,
```